### ISSUE TITLE
Losing items on death, expected or not?

### ISSUE THREAD

**Issue by jaded33** [B1n-ry/Youre-in-grave-danger#12]
Not sure if a bug or expected feature but my son and I have lost an item or two when grabbing our items from grave after death, in comparison to the inventories we had before death.  It is usually a sword that goes missing, possibly because it is a hand held item at the time of death.  Hard to track what other items may or may not go missing, but usually feels like we got most of our stuff back from the grave but missing a sword and maybe one other thing.  Thought maybe it was a set penalty for death but didn't see a config for that only for XP.  (Un)fortunately I haven't died in a bit but my son is reporting he is still losing an item or two. 
So just wondering if we lose our hand held items at death or is this expected?  No trinkets items being used. 

**Expected behavior**

**Desktop (please complete the following information):**
 - OS: Windows and MacOS clients, linux self-hosted local server.
 - Minecraft Version: 1.18
 - Version: 1.1.11

**Comment by B1n-ry** [comment]
No this is not a feature of the mod. It's a bug. I've had this bug happen to me in my testing and I though I fixed it, but I guess not. I'll be trying to fix this issue within the next few hours. Also if you do find something that should be configurable within the mod, please make a feature request for that. Won't add *this* as a config however, since as you say it's not exactly easy to track which item it is. (I will if you specificly request it though)
I'll reply here again as soon as it's fixed. Thanks for reporting the bug!

**Comment by B1n-ry** [comment]
UPDATE: This issue was apparently harder than I thought to replicate. Which version are you playing? You've stated that you play one of the 1.17 versions (1.1.11) in 1.18. The latest 1.18 release is 1.2.5. Please make sure your server mod version is also up to date. In my testing I've tried multiple ways of dying with various inventory layouts and nothing has disappeared. If this is not just because it's an old version, if possible, please try and recreate your sons inventory before he died, and when the inventory equipped before retrieving the grave. If this is not due to an old version, I need screenshots of the conditions of this issue taking place. I'm sorry you're have this issue. I'll try to the best of my abilities to help you

**Comment by jaded33** [comment]
LOL it was 1.2.4 version when I wrote the bug report I quickly checked your last release version prior to the update on your tags here.  It was like 2AM for me.  Verified now I was on 1.2.4  I have updated client and server now to 1.2.5 and I will ask my son and daughter to try and keep track if we lose anything next death.  I try not to die but my son literally kamikazes himself.  

Thanks for your reply I have used a few death mods in the past but liked yours for your desire to have a complete full config and options.
